src/site/content/en/blog/performance-optimizing-content-efficiency-client-hints/index.md

```diff
@@ -0,0 +1,722 @@
+
+You can set these opt-in headers in any back-end language. For example, [PHP’s
+`header` function](http://php.net/manual/en/function.header.php) could be used.
+You could even set these opt-in headers with [the `http-equiv`
+attribute](https://developer.mozilla.org/docs/Web/HTML/Element/meta#attr-http-equiv)
+on a `<meta>` tag:
+
+```html
+<meta http-equiv="Accept-CH" content="Viewport-Width, Downlink" />
+```
+
+{% Aside 'warning' %}
+Earlier versions of the spec included the `Accept-CH-Lifetime` header
+which has now been removed and sites should also stop sending it. If you were
+using this to enable hints to persist over browser sessions you should instead
+ensure you are sending the necessary `Accept-CH` headers on the relevant
+responses.
+{% endAside %}
+
+## All the client hints!
+
+Client hints describe one of two things: the device your users, well, _use_, and
+the network they’re using to access your site. Let’s briefly cover all of the
+hints that are available.
+
+### Device hints
+
+Some client hints describe characteristics of the user’s device, usually screen
+characteristics. Some of them can help you choose the optimal media resource for
+a given user’s screen, but not all of them are necessarily media-centric.
+
+Before we get into this list, it might be helpful to learn a few key terms used
+to describe screens and media resolution:
+
+**Intrinsic size:** the actual dimensions of a media resource. For example, if
+you open an image in Photoshop, the dimensions shown in the image size dialogue
+describe its _intrinsic size_.
+
+**Density-corrected intrinsic size:** the dimensions of a media resource after
+it has been corrected for pixel density. It’s the image’s _intrinsic size_
+divided by a [device pixel
+ratio](https://developer.mozilla.org/docs/Web/API/Window/devicePixelRatio).
+For example, let’s take this markup:
+
+```html
+<img
+  src="whats-up-1x.png"
+  srcset="whats-up-2x.png 2x, whats-up-1x.png 1x"
+  alt="I'm that image you wanted."
+/>
+```
+
+Let’s say the intrinsic size of the `1x` image in this case is 320x240, and the
+`2x` image’s intrinsic size is 640x480. If this markup is parsed by a client
+installed on a device with a screen device pixel ratio of 2 (e.g., a Retina
+screen), the `2x` image is requested. The _density-corrected intrinsic size_ of
+the `2x` image is 320x240, since 640x480 divided by 2 is 320x240.
+
+**Extrinsic size:** the size of a media resource after CSS and other layout
+factors (such as `width` and `height` attributes) have been applied to it. Let’s
+say you have an `<img>` element that loads an image with a density-corrected
+intrinsic size of 320x240, but it also has CSS `width` and `height` properties
+with values of `256px` and `192px` applied to it, respectively. In this example,
+the _extrinsic size_ of that `<img>` element becomes 256x192.
+
+<figure>
+{% Img src="image/C47gYyWYVMMhDmtYSLOWazuyePF2/mYLOc0CZtVxXCNDoHcsK.svg", alt="An illustration of intrinsic size versus
```

Comment:
I think the quotes in the alt are what the linter is complaining about.

Reply:
@rachelandrew Thank you, I've removed the quotes from the images